apps/core/mutations.py

```diff
@@ -66,6 +67,97 @@ class Arguments:
     result = graphene.Field(TableType)
 
 
+class UpdateTableProperties(graphene.Mutation):
+    class Arguments:
+        data = TablePropertiesInputType(required=True)
+        id = graphene.ID(required=True)
+
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    ok = graphene.Boolean()
+    result = graphene.Field(TableType)
+
+    @staticmethod
+    def mutate(root, info, id, data):
+        try:
+            instance = Table.objects.get(id=id)
+        except Table.DoesNotExist:
+            return DeleteTableFromWorkspace(
```

Comment:
Should this not be `UpdateTableProperties`? `DeleteTableFromWorkspace` seems like typo for me.

Reply:
Done